integration-tests/spring-extension-support-test/src/main/java/com/example/application/autogrid/PropertyStringFilterSpecification.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2023 Marco Collovati, Dario Götze
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.application.autogrid;
+
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Expression;
+import jakarta.persistence.criteria.Path;
+import jakarta.persistence.criteria.Predicate;
+import jakarta.persistence.criteria.Root;
+import java.sql.Date;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+
+import dev.hilla.crud.filter.PropertyStringFilter;
+
+/*
+ * NOTE: this code has been copy/pasted from Hilla code base, credit goes to Vaadin Ltd
+ * https://github.com/vaadin/hilla/blob/main/packages/java/endpoint/src/main/java/dev/hilla/crud/PropertyStringFilterSpecification.java
+ *
+ * Copy/paste is required to remove the Spring Specification<T> interface, not supported by quarkus-spring-data extension
```

Comment:
```suggestion
 * Adaptation is required to remove the Spring Specification<T> interface, not supported by quarkus-spring-data extension
```
Use same wording, or is this really just copy/paste?